### ISSUE TITLE
Drag and drop freeze with multitouch

### ISSUE THREAD

**Issue by MDB73** [fullcalendar/fullcalendar#5691]
### Reduced Test Case

not reproducible with a link

### Bug Description

Fullcalendar v5.2.0. In the "timeGridDay" view, if you use multi-touch for drag and drop, a mirror event is rendered, but it is blocked and no longer allows you to perform other functions.

### Screenshots

![Screenshot_20200802_204925 1](https://user-images.githubusercontent.com/36129038/89130001-ff1d1100-d501-11ea-905d-3330ce271823.jpg)

**Comment by acerix** [comment]
Do you get the same issue in this demo after switching to day view?  Also, which device and how are you multi-touching exactly?

https://fullcalendar.io/docs/event-dragging-resizing-demo

I tested on Android and it works as expected without freezing.  There is no multi-touch support, it moves the event based only on one of the "touches".

**Comment by MDB73** [comment]
As you can see from the screenshot, unfortunately yes. The issue is the same. You can reproduce the problem by putting two finger, at the same time, on the event.

I know there is no multi touch support. I tried to solve the problem independently but unfortunately I was unable to find the solution. I use 
I tried with a tablet (chuwi) Android 9 (Firefox and Chrome). Also with a SAMSUNG S8 Android 9 (Firefox, Chrome and Samsung Browser).

![Screenshot_20200805_104801 1](https://user-images.githubusercontent.com/36129038/89392124-6658e200-d709-11ea-82dc-3472b37ace56.jpg)

**Comment by acerix** [comment]
Thanks for the details, it looks like a bug from your screenshot although I still couldn't reproduce the issue on my device.

**Comment by MDB73** [comment]
It's very strange. Regardless of the device I tested with, the problem has always occurred. If you want I can make a video in which I show what happens on different devices.

**Comment by arshaw** [comment]
@MDB73 could you please test this with the latest v5.3.0 ? I made some touch-related fixes in there. Might have fixed this also (tho I still have not reproduced this)

**Comment by MDB73** [comment]
Unfortunately the problem still exists. I made a video to show you what happens. No matter what device you use the problem always occurs. I also asked my friends to try using their devices and they also encountered the same problem as me.

https://streamable.com/98mt9q

**Comment by arshaw** [comment]
I see you are using Firefox on Android. Does the same problem happen on Chrome on Android?

**Comment by MDB73** [comment]
Yes, also with Chrome and Samsung Browser on Android. This is the video with Chrome:
https://streamable.com/vdllqx

**Comment by MDB73** [comment]
I finally solved this little problem on my own. 
Whenever a finger is placed on an event, the "handleTouchStart" method is triggered. So if we use two fingers at the same time this creates a problem.

The solution is simple just add "if (this.isTouchDragging === true) return" to the "PointerDragging.ts" file before line number 139.

Once this is done the code should become like this:

handleTouchStart = (ev: TouchEvent) => {
    if (this.isTouchDragging === true) return
    if (this.tryStart(ev)) {
      this.isTouchDragging = true

**Comment by acerix** [comment]
Related issue reported in https://fullcalendar.freshdesk.com/a/tickets/2705 likely will also be fixed by this.  

Using multi-touch to scroll while dragging an event resets the scroll position.
